package.json

```diff
@@ -58,7 +59,7 @@
     "typescript": "^4.5.3"
   },
   "peerDependencies": {
-    "constructs": "^3.3.165"
+    "constructs": "3.x - 10.x"
```

Comment:
TIL about this syntax. Also I am not sure that's what we want, because theoretically, 4.x, 5.x is not supported.

So I wonder if we want this to be `^3 || ^10` or `3.x || 10.x`

Reply:
TIL

Can we document somewhere?